### PR TITLE
chore(wip): audit css

### PR DESCRIPTION
**Pull request by cathysiller**
JIRA: SURF-1169

### LGTM's
- [x] Dev LGTM
- [x] Zoom LGTM

### CODE REVIEW

**Comment by HelixUI** [comment]
Deploy preview for *helix-ui* processing.

Built with commit 7c318c18628ede1f47b5a45acd53b0ce64110c34

https://app.netlify.com/sites/helix-ui/deploys/5bb3db0c73f2cf31cb9bed2d

File: docs/docs.less [hunk 1]
```diff
@@ -90,6 +86,7 @@ main {
 }
 
 // Component Status Styles
+// TODO: remove block
```
Comment (CITguy) [inline docs/docs.less]: Remove comment

File: docs/docs.less [hunk 2]
```diff
@@ -265,12 +262,14 @@ dl:not(.hxList) {
   }
 }
 
+// currently being used in React compatibility guide
```
Comment (CITguy) [inline docs/docs.less]: Remove comment

File: docs/docs.less [hunk 3]
```diff
@@ -265,12 +262,14 @@ dl:not(.hxList) {
...
 iframe[src*="codepen.io"] {
   border: 2px solid @gray-900;
   height: 25rem;
   width: 100%;
 }
 
+// currently being used in polyfills guide
```
Comment (CITguy) [inline docs/docs.less]: Remove comment

File: docs/stability.html [hunk 4]
```diff
@@ -1,3 +1,4 @@
+{# TODO: remove file #}
```
Comment (CITguy) [inline docs/stability.html]: Remove comment

File: src/helix-ui/styles/components/alerts.less [hunk 5]
```diff
@@ -1,3 +1,4 @@
+// TODO: move to hx-alert.less
```
Comment (CITguy) [inline src/helix-ui/styles/components/alerts.less]: Remove comment

**Pull request merged**